assets/src/scripts/components/hydrograph/domain.js

```diff
@@ -52,12 +54,10 @@ export const extendDomain = function (domain, lowerBoundPOW10) {
 };
 
 
-export const getYDomain = function (pointArrays, currentVar) {
+export const getYDomain = function (pointArrays, currentVarParmCd) {
     let yExtent;
     let scaleDomains = [];
 
-    let currentVarParmCd = currentVar && currentVar.variableCode ? currentVar.variableCode.value : null;
-
```

Comment:
This stuff you did looks good to simplify things!

It also brings to mind a question I've had for awhile... are you aware of if there are any circumstances non-USGS parameter codes overlapping with USGS ones?

Reply:
parameter codes are a USGS-specific thing.  Other agencies may have parameter codes(EPA used to use them), but there is no authority management.

Reply:
Yes - so WaterML has that distinction, between variables (globally unique) and variableCodes (organizationally unique). I'm wondering if we have any chance of the database having organizationally unique code overlap? How many outside agency sites does the DB even have?

Reply:
For the purposes of this question, none.  There are sites managed by USGS that are paid for by other agencies- that's generally where the agency code comes from- but they are otherwise indistinguishable from site with USGS as the agency code